src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2477,6 +2522,42 @@
         </TextBlock>
     </DataTemplate>
     
+
+    <Style TargetType="Expander" x:Key="InstalledPackagesExpanderStyle">
+        <Setter Property="Template">
+            <Setter.Value>
+                <!-- Control template for expander -->
```

Comment:
I notice you're not using the header template:
https://github.com/DynamoDS/Dynamo/pull/11750/files#diff-094c8df98e6724d704bb015c259cdf5352917a2076c57a4b4ffccaa74d003c4dR2564

that I added to the other two expander styles, that means the fontsize (and eventually font) of the header will not be easily changed for all expanders - can we use it or does it interfere with some other requirement?

Reply:
I am not sure if there'll be an issue. 
If I just add the template specifier in the style definition, then the control gets messed up
Maybe I just don't know how to use it...

Reply:
It does not work for us because we have a `Grid` at the top level, not a `TextBlock`.

Reply:
It does not work because we have a `Grid` in our header, not a `TextBlock`?

Reply:
Apologies - I didn't realize this expander was actually embedded into the installed packages view - I don't think we would want to use this template anyway. Forget the above comment.